meorg_client/client.py

```diff
@@ -25,84 +26,128 @@ def __init__(self, base_url=mcc.MEORG_URL, email=None, password=None):
         password : str, optional
             User password, by default None
         """
-        self.base_url = base_url
-        self.headers = dict()
+
+        # Initialise the mimetypes
+        mt.init()
+
+        # Ensure there is a trailing slash
+        self.base_url = f"{base_url}/" if base_url[-1] != "/" else base_url
+        self.headers = {"Cache-Control": "no-cache", "Pragma": "no-cache"}
         self.last_response = None
 
-        # Automatically login.
+        # Automatically login if credentials are set.
         if email is not None and password is not None:
             self.login(email, password)
```

Comment:
Maybe a warning/log that email/password isn't set?

Reply:
Not super necessary. There may be situations where the class will need to be instantiated, modified and then the user would log in. Legitimate use case that doesn't constitute a warning.